go.mod

```diff
@@ -83,7 +84,7 @@ require (
 	github.com/golang/protobuf v1.5.3 // indirect
 	github.com/golang/snappy v0.0.4 // indirect
 	github.com/google/gofuzz v1.2.0 // indirect
-	github.com/google/uuid v1.3.1 // indirect
+	github.com/google/uuid v1.3.1
```

Comment:
should this move up to the direct dependencies then?

Reply:
this is being addressed separately in https://github.com/prometheus-operator/prometheus-operator/pull/6114